onnxruntime/core/providers/cpu/math/element_wise_ops.cc

```diff
@@ -562,10 +562,80 @@ struct Min_8::ComputeImpl {
   }
 };
 
+template <bool is_min>
+static Status MinMaxMLFloat16(const OpKernel& inst, OpKernelContext* context) {
```

Comment:
Use as param instead of template?

Reply:
If we use a param, we have to incur the runtime check every time each of the lambdas are being invoked. This way, it incurs a small code duplication but atleast there is no runtime check every time each lambda is being invoked (and this could add up)